packages/web/src/components/premium-content-purchase-modal/components/PayExtraFormSection.tsx

```diff
@@ -22,10 +22,12 @@ const formatPillAmount = (val: number) => `$${Math.floor(val / 100)}`
 
 export type PayExtraFormSectionProps = {
   amountPresets: PayExtraAmountPresetValues
+  disabled?: boolean
 }
 
 export const PayExtraFormSection = ({
-  amountPresets
+  amountPresets,
+  disabled = false
```

Comment:
nit: I don't think you need to default initialize a boolean to false as pretty much any if() check against undefined will result in what you want.

(Also, purely academic but the "HTML element behavior" we are emulating is usually `<button disabled />` for true and `<button />` for false, so pass an undefined value instead of explicitly false)

Reply:
great call ty